### PR TITLE
Gemini 0.18.0 WIP

### PR DESCRIPTION
**Pull request by TorHou**
I hope to finish the other tools tomorrow. 

Also the dependencies have to be fixed:
package_gemini
and the datamanager for the gemini dbs.

### CODE REVIEW

File: tools/gemini/gemini_load.xml [hunk 1]
```diff
@@ -10,11 +10,13 @@
     <command>
 <![CDATA[
         ln -s "${ infile }" input.vcf &&
+        bgzip -c input.vcf > input.vcf.gz &&
+        tabix -p vcf input.vcf.gz &&
 
         gemini
-            --annotation-dir ${annotation_databases.fields.path}
+            --annotation-dir "/home/houwaart/data/gemini/gemini_data"
```
Comment (bgruening) [inline tools/gemini/gemini_load.xml]: These things we need to remember :)

File: tools/gemini/gemini_query.xml [hunk 2]
```diff
@@ -72,8 +72,7 @@
 
         <expand macro="add_header_column" />
 
-        <!-- TODO: is there any default values set? -->
-        <param name="min_kindreds" type="integer" value="-1" label="Minimum number of families for a variant passing a family-wise filter to be in" help="-1 means default values (--min-kindreds)" />
+        <expand macro="min_kindreds" />
```
Comment (bgruening) [inline tools/gemini/gemini_query.xml]: I love macros!

**Comment by bgruening** [comment]
Thanks @TorHou!
I will try to work on the dependencies ....

**Comment by bgruening** [comment]
Let me know when you feel I can merge this.

**Comment by TorHou** [comment]
I will. Cheers!

**Comment by TorHou** [comment]
The other tools still need tests. Will update soon.

**Comment by bgruening** [comment]
Awesome! Merry Christmas!

**Comment by TorHou** [comment]
I think it's ready for merging.
Merry Christmas :christmas_tree:

**Comment by bgruening** [comment]
Awesome! So we need to fix the tool_dependencies.xml and the datamanager right?

**Pull request merged**